examples/java8/pom.xml

```diff
@@ -75,6 +75,12 @@
             <goals><goal>analyze-only</goal></goals>
             <configuration>
               <failOnWarning>true</failOnWarning>
+              <ignoredUnusedDeclaredDependencies>
+                <!-- runners are used by ServiceLoader. -->
```

Comment:
Not sure I understand this.

For example, you might want to add dependencies in the `runtime` scope.